### ISSUE TITLE
[BUG] kube-linter v0.1.6 does not respect file based configuration properties.

### ISSUE THREAD

**Issue by AlanMasciangelo** [stackrox/kube-linter#149]
**System info:**
 - OS: Windows

**Describe the bug**

kube-linter v0.1.6 does not respect file based configuration properties.

**To Reproduce**

Create a yaml config such as the following from the [disabling all default checks example](https://docs.kubelinter.io/#/configuring-kubelinter?id=disable-all-default-checks)

```yaml
checks:
  doNotAutoAddDefaults: true
```

**Expected behavior**

```bash
$ kube-linter.exe lint my-charts/ --config kubelinter-config.yaml
Warning: no checks enabled.
```

**Screenshots**

No screenshot available but the deafult checks do run although I expected them not to.

**Additional context**

I do not have this problem with 0.1.4 or 0.1.5 using the exact same config file.  I do see marshalling errors if I pass a bad config in. CLI arguments suchas `--do-not-auto-add-defaults` work as expected.

**Comment by viswajithiii** [comment]
Thanks for filing, @AlanMasciangelo!
